ui/src/app/app.tsx

```diff
@@ -170,6 +170,12 @@ export class App extends React.Component<{}, { popupProps: PopupProps, error: Er
                             <Redirect path='*' to='/'/>
                         </Switch>
                     </Router>
+                    <div style={{position: 'fixed', left: 70, bottom: 5}}>
```

Comment:
Could you please render UI version next to Argo logo and add tooltip which shows both UI and backend version:

![Screen Shot 2019-10-18 at 1 09 25 PM](https://user-images.githubusercontent.com/426437/67124945-87673b00-f1a8-11e9-98ef-7ad720b2e1b0.png)